packages/zone/src/durable.js

```diff
@@ -113,5 +113,3 @@ export const makeDurableZone = (baggage, baseLabel = 'durableZone') => {
   });
 };
 harden(makeDurableZone);
-
-export { M };
```

Comment:
Well, I found out why --- `M` is imported from `@agoric/zone`. Rather than restore this, which seems very weird to me, instead I fixed the importers to import `M` from `@endo/patterns` instead.

If you don't like this, feel free to instead restore the exporting of `M` from `@agoric/zone`. Though, perhaps better to export it from the exporting files (index.js, etc) rather than exporting it from the substantive source code files.